package-lock.json

```diff
@@ -32134,7 +32135,9 @@
       "dev": true
     },
     "webpack": {
-      "version": "5.75.0",
```

Comment:
Question: why are these changes not updates in `package.json`

Reply:
Because I used `npm update` over `npm install`.

Both things work :).